### PR TITLE
Adds an Athena client

### PR DESCRIPTION
**Pull request by colestrode**
I also reorganized the README to make the documentation easier to find and read.

### CODE REVIEW

File: lib/athena/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        {}
+      );
+    }
+
+    //the first row that comes back is the column names, so don't include that
+    return _.map(_.tail(Rows), formatRow);
+  }
+}
+
+module.exports = function getClient(opts) {
```
Comment (James-Burba) [inline lib/athena/index.js]: Can we include a jsdoc comment with the expected params (database, s3bucket)? Some IDEs will show that, and it's easier than digging through the docs.

File: lib/athena/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Athena
+
+A simplified, promise-based library for querying [Athena](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Athena.html). This removes the need write custom polling logic.
+
+## Get a client
+
+```
+const athenaClient = new require('@sparkpost/aws').Athena( opts )
```
Comment (James-Burba) [inline lib/athena/README.md]: As someone still pretty new to javaescript, is it common to use `new` when we're not calling a class constructor? The function the module exports just returns an object with a property that's a function.

Comment (colestrode) [reply lib/athena/README.md]: good call, there's no need for the `new`. That's just a hold over from how I was thinking of implementing it.

File: lib/athena/index.js (hunk 1 above)
Comment (colestrode) [reply lib/athena/index.js]: sure thing!

**Pull request closed without merging**